recipes/packio/all/conandata.yml

```diff
@@ -44,3 +44,6 @@ sources:
   "2.5.0":
     url: "https://github.com/qchateau/packio/archive/2.5.0.tar.gz"
     sha256: "5aa29313ac37be2fb66ff01af9413d9f9931cd4566f1400b00fac107552558ae"
+  "2.6.0":
```

Comment:
This is failing on CI because the recipe (even earlier versions) is trying to modify the value of an option in the `configure()` method 

```
======== Computing dependency graph ========
ERROR: packio/2.6.0: Error in requirements() method, line 67
	self.options.boost_json = not self.options.standalone_asio
	ConanException: Incorrect attempt to modify option 'boost_json' from 'default' to 'True'
```

(cc @ErniGH, @AbrilRBS : this needs to be refactored)